core/lib.rs

```diff
@@ -45,3 +46,5 @@ pub fn v8_version() -> &'static str {
 fn test_v8_version() {
   assert!(v8_version().len() > 3);
 }
+
+crate_modules!["test_file.ts"];
```

Comment:
Would be nice if it wasn't required to list them like this. Not sure if that's possible to avoid tho.

Reply:
Could `crate_modules!`/`include_crate_modules!` return path to the installed crate? That way, we could easily resolve arbitrary files from crate, without needing to specify them explicitly.

Reply:
@bartlomieju 👍 I like that better. Still don't think it's perfect, but it makes things much simpler.